app/src/main/java/com/sun/basic_japanese/util/MapConverter.kt

```diff
@@ -0,0 +1,13 @@
+package com.sun.basic_japanese.util
+
+object MapConverter {
+    fun <K, V> combine(keys: List<K>, values: List<V>): List<Pair<K, V>> {
+        val result = hashMapOf<K, V>()
+        val keyIterator = keys.iterator()
+        val valueIterator = values.iterator()
+        while (keyIterator.hasNext()) {
```

Comment:
```suggestion
        while (keyIterator.hasNext() && valueIterator.hasNext()) {
```
Nên check cả 2 để tránh trường hợp 1 trong 2 list ngắn hơn sẽ bị crash